src/api/tcr/TcrApiProxy.ts

```diff
@@ -0,0 +1,14 @@
+import { TcrApiImpl, Params, TcrApi } from './TcrApi'
+import { CacheMixin } from 'api/proxy'
+
+export class TcrApiProxy extends TcrApiImpl {
+  private cache: CacheMixin
+
+  public constructor(params: Params) {
+    super(params)
+
+    this.cache = new CacheMixin()
+
+    this.cache.injectCache<TcrApi>(this, [{ method: 'getTokens' }])
```

Comment:
This is great! I love how you always add the caches for the APIS 🎸

Reply:
:man_singer: